infinite/index.ts

```diff
@@ -225,7 +220,13 @@ export const infinite = (<Data, Error>(useSWRNext: SWRHook) =>
           }
         }
 
-        return args.length ? swr.mutate(data, shouldRevalidate) : swr.mutate()
+        // We are using `arguments` here because there's no other way around to
+        // tell if an argument is missing or `undefined`. It will be compiled to
+        // `arguments` anyways.
+        // @ts-ignore
+        return arguments.length
```

Comment:
Can we use casting for arguments instead of ts-ignore comment?

Reply:
> casting for arguments

Can you please explain it?